src/worker/connectionincoming.js

```diff
@@ -151,8 +151,40 @@ class ConnectionIncoming {
         this.state.realname = upstream.state.realname;
 
         let nick = this.state.nick;
+
+        // Let plugins modify/add isupport tokens
+        let isupports = upstream.state.registrationLines.filter((regLine) => regLine[0] === '005');
+        let isupportTokens = isupports.reduce((prev, cur) => {
+            // Remove the last token if it contains a space. Usually 'is supported by this server'.
```

Comment:
Not gonna cause any issues, but as a note the space check may mean that if the server outputs a localised translation for _"is supported by this server"_ (that includes no spaces, e.g. an Asian language or something) then we'd interpret the last param as a token. I think every server I've seen outputs the last param as the human-readable description thing.